examples/istio/quick_start.yaml

```diff
@@ -162,11 +162,14 @@ data:
   inject.rego: |
     package istio
 
+    uid := input.request.uid
+
     inject = {
-      "apiVersion": "admission.k8s.io/v1beta1",
+      "apiVersion": "admission.k8s.io/v1",
       "kind": "AdmissionReview",
       "response": {
         "allowed": true,
+        "uid": uid,
```

Comment:
Needed to include the uid for the same reason as the issue below.
https://github.com/open-policy-agent/opa/issues/2467